java/ql/test/library-tests/frameworks/spring/util/test.ql

```diff
@@ -0,0 +1,71 @@
+import java
+import semmle.code.java.dataflow.DataFlow
+import semmle.code.java.dataflow.ExternalFlow
+import semmle.code.java.dataflow.TaintTracking
+import TestUtilities.InlineExpectationsTest
+import semmle.code.java.frameworks.spring.SpringUtil
```

Comment:
Remove. `ExternalFlow` needs bidirectional import, so `SpringUtil` should be imported there instead.

Reply:
At the moment it seems like the Spring models are not imported by default, which is why I didn't add the import anywhere---I guess the taint flow steps should always be included?

Reply:
Yes. Otherwise they don't actually accomplish anything - as you'll see in this test if you remove this line.

Reply:
Right, but that's true of all the Spring models, as far as I can tell? I have included this in `Spring.qll`.